src/DataFramesMeta.jl

```diff
@@ -435,38 +451,64 @@ Sort by criteria. Normally used to sort groups in GroupedDataFrames.
 ```jldoctest
 julia> using DataFrames, DataFramesMeta, Statistics
 
-julia> d = DataFrame(n = 1:20, x = [3, 3, 3, 3, 1, 1, 1, 2, 1, 1,
-                                    2, 1, 1, 2, 2, 2, 3, 1, 1, 2]);
+julia> d = DataFrame(x = [3, 3, 3, 2, 1, 1, 1, 2, 1, 1], n = 1:10);
+
+julia> @orderby(d, -1 .* :n)
+10×2 DataFrame
+│ Row │ x     │ n     │
+│     │ Int64 │ Int64 │
+├─────┼───────┼───────┤
+│ 1   │ 1     │ 10    │
+│ 2   │ 1     │ 9     │
+│ 3   │ 2     │ 8     │
+│ 4   │ 1     │ 7     │
+│ 5   │ 1     │ 6     │
+│ 6   │ 1     │ 5     │
+│ 7   │ 2     │ 4     │
+│ 8   │ 3     │ 3     │
+│ 9   │ 3     │ 2     │
+│ 10  │ 3     │ 1     │
 
 julia> g = groupby(d, :x);
 
-julia> @orderby(g, mean(:n))
-GroupedDataFrame  3 groups with keys: Symbol[:x]
-First Group:
-5×2 SubDataFrame{Array{Int64,1}}
-│ Row │ n  │ x │
-├─────┼────┼───┤
-│ 1   │ 1  │ 3 │
-│ 2   │ 2  │ 3 │
-│ 3   │ 3  │ 3 │
-│ 4   │ 4  │ 3 │
-│ 5   │ 17 │ 3 │
-⋮
-Last Group:
-6×2 SubDataFrame{Array{Int64,1}}
-│ Row │ n  │ x │
-├─────┼────┼───┤
-│ 1   │ 8  │ 2 │
-│ 2   │ 11 │ 2 │
-│ 3   │ 14 │ 2 │
-│ 4   │ 15 │ 2 │
-│ 5   │ 16 │ 2 │
-│ 6   │ 20 │ 2 │
+julia> @linq g |>
+       transform(t = :n .- mean(:n)) |>
+       orderby(:n .- mean(:n))
```

Comment:
I do not understand this example. I would assume that the result of `transform` is a `DataFrame` and then you essentially sort it by `:n` (it was already sorted by `:n` actually, so here `@orderby` is no-op). But how does it give us an example how `@orderby` works on `GroupedDataFrame`?

Reply:
Here is an extra sentence I added to the dosctring 

> Note that the column `:t` is arranged from
lowest to highest after the `@orderby` command. This shows that
`@orderby` is equivelent to a transformation by group followed
by ordering on the subsequent transformation.

Does this explanation make sense?